core/src/main/java/me/mykindos/betterpvp/core/command/commands/admin/SearchCommand.java

```diff
@@ -238,12 +238,25 @@ public String getDescription() {
         public void execute(Player player, Client client, String... args) {
             clientManager.sendMessageToRank("Search", UtilMessage.deserialize("<yellow>%s</yellow> is retrieving all <light_purple>UUIDitems</light_purple> currently being held", player.getName()), Rank.HELPER);
 
+            boolean atLeastOneUUIDItemFound = false;
             for (Player online : Bukkit.getOnlinePlayers()) {
-                Component component = UtilMessage.deserialize("<yellow>%s</yellow> is holding:", online.getName());
+                boolean hasAtLeastOneUUIDItem = false;
+                Component component = UtilMessage.deserialize("<yellow>%s</yellow> is holding:", online.getName());;
```

Comment:
extra ; at the end as pointed out by sonar

Reply:
```suggestion
                Component component = UtilMessage.deserialize("<yellow>%s</yellow> is holding:", online.getName());
```